scripts/spack/packages/tribol/package.py

```diff
@@ -0,0 +1,299 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from spack.spec import UnsupportedCompilerError
+
+import os
+import socket
+from os.path import join as pjoin
+
+
+def get_spec_path(spec, package_name, path_replacements={}, use_bin=False):
+    """Extracts the prefix path for the given spack package
+       path_replacements is a dictionary with string replacements for the path.
+    """
+
+    if not use_bin:
+        path = spec[package_name].prefix
+    else:
+        path = spec[package_name].prefix.bin
+
+    path = os.path.realpath(path)
+
+    for key in path_replacements:
+        path = path.replace(key, path_replacements[key])
+
+    return path
+
+# NOTE: Cannot inherit from builtin until tribol becomes a builtin spack package within spack itself.
+
+class Tribol(CachedCMakePackage, CudaPackage):
+    """Tribol is an interface physics library."""
+
+    homepage = "https://github.com/LLNL/Tribol"
+    git      = "https://github.com/LLNL/Tribol.git"
+
+    version("develop", branch="develop", submodules=True)
+
+
+    # SERAC EDIT START
+    # Note: We add a number to the end of the real version number to indicate that we have
+    #  moved forward past the release. Increment the last number when updating the commit sha.
+    version('0.1.0.5', commit='c62690d8c21f94110d1f97d50c89fa87d142bb59', submodules=True, preferred=True)
```

Comment:
Would it be possible to point to this commit for now: [`3ed15810a8cf9cebda84f97ee35fa5ee88154fc5`](https://github.com/LLNL/Tribol/pull/11/commits/3ed15810a8cf9cebda84f97ee35fa5ee88154fc5).  It's still part of a Tribol PR, but it has the interface that I'll be hooking up to serac.